data_handler/handlers/loan_states/hashtack_v0/interest_rate.py

```diff
@@ -54,102 +55,98 @@ def _set_events(self, start_block: int, end_block: int) -> None:
             supply_apr_bps = Decimal(int(event["data"][2], 16))
 
             # Calculate interest rate for supply and borrow and convert to percents using the formula:
             # (apr * seconds_passed / seconds_in_year) / 10000
-            current_collateral_change = (supply_apr_bps * seconds_passed / SECONDS_IN_YEAR) * percents_decimals_shift
-            current_debt_change = (borrow_apr_bps * seconds_passed / SECONDS_IN_YEAR) * percents_decimals_shift
+            cumulative_collateral_interest_rate_increase = (
+                    (supply_apr_bps * seconds_passed / SECONDS_IN_YEAR) * percents_decimals_shift
+            )
+            cumulative_debt_interest_rate_increase = (
+                    (borrow_apr_bps * seconds_passed / SECONDS_IN_YEAR) * percents_decimals_shift
+            )
             interest_rate_state.update_state_cumulative_data(
-                token_name, event["block_number"], current_collateral_change, current_debt_change
+                token_name, event["block_number"],
+                cumulative_collateral_interest_rate_increase,
+                cumulative_debt_interest_rate_increase,
             )
 
         # Write last block data
-        self._add_block_data(blocks_data, interest_rate_state.build_interest_rate_model(HASHSTACK_ID))
-        return blocks_data
+        self._add_block_data(interest_rate_state.build_interest_rate_model(HASHSTACK_ID))
 
-    def _get_blocks_bounds(self, start_block, end_block, latest_block) -> tuple[int, int]:
+    def _get_blocks_bounds(self, end_block: int, latest_block: int) -> tuple[int, int]:
         """
         Calculate the bounds for the blocks pagination.
-        :param start_block: int - Previous start block number.
         :param end_block: int - Previous end block number.
         :param latest_block: int - The latest block number.
         :return: tuple[int, int] - The new start and end block numbers.
         """
         if end_block + self.PAGINATION_SIZE < latest_block:
-            start_block += end_block
+            return end_block, end_block + self.PAGINATION_SIZE
         else:
-            end_block = latest_block
-            return start_block, end_block
-        if start_block + self.PAGINATION_SIZE <= latest_block:
-            end_block += self.PAGINATION_SIZE
-        else:
-            end_block = latest_block
-        return start_block, end_block
+            return end_block, latest_block
 
     async def _run_async(self) -> None:
         """Asynchronous function for running the interest rate calculation process and fetching data from on-chain."""
         latest_block = await NET.get_block_number()
-        start_block = self.last_block_data.block if self.last_block_data else 0
-        if start_block == latest_block:
+        last_block = self.last_block_data.block if self.last_block_data else 222000
+        if last_block == latest_block:
             return
         start_block, end_block = self._get_blocks_bounds(
```

Comment:
It would make the code a lot more readable if you specified the parameter names. Now I have to check the function definitions many times to understand the code.